$OpenBSD: patch-cmake_modules_KDE4Macros_cmake,v 1.5 2013/04/24 19:45:29 zhuk Exp $
Target name with spaces is not a very good idea, kalarm in kdepim breaks.
--- cmake/modules/KDE4Macros.cmake.orig	Thu Jan 24 01:44:16 2013
+++ cmake/modules/KDE4Macros.cmake	Wed Feb  6 22:37:25 2013
@@ -1350,7 +1350,7 @@ function(KDE4_INSTALL_AUTH_ACTIONS HELPER_ID ACTIONS_F
                        WORKING_DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}
                        COMMENT "Generating ${HELPER_ID}.policy"
                        DEPENDS ${_KDE4_KAUTH_POLICY_GEN_EXECUTABLE_DEP})
-    add_custom_target("actions for ${HELPER_ID}" ALL DEPENDS ${_output})
+    add_custom_target(kauth-actions-for-${HELPER_ID} ALL DEPENDS ${_output})
 
     install(FILES ${_output} DESTINATION ${KDE4_AUTH_POLICY_FILES_INSTALL_DIR})
   endif()
